app-common/src/main/kotlin/net/thunderbird/app/common/account/AppCommonAccountModule.kt

```diff
@@ -43,10 +48,16 @@ internal val appCommonAccountModule = module {
         )
     }
 
+    factory<ImmutableList<Int>>(named("AccountColors")) {
+        androidContext().resources.getIntArray(
+            ThemeCommonR.array.account_colors,
+        ).toList().toImmutableList()
+    }
+
     factory {
         AccountColorPicker(
-            accountManager = get(),
-            resources = get(),
+            repository = get(),
+            accountColors = get(named("AccountColors")),
```

Comment:
suggestion: use a const to `"AccountColors"`, so it makes easier to maintain in the feature.

```suggestion
            accountColors = get(named(AccountColorPicker.ACCOUNT_COLORS_QUALIFIER)),
```

and then:
```kotlin
class AccountColorPicker(/* ... */) {
    companion object {
        const val ACCOUNT_COLORS_QUALIFIER = "AccountColors"
    }
    // ... rest of the code
}

```